dataline-scheduler/src/main/java/io/dataline/scheduler/WorkerRun.java

```diff
@@ -96,11 +97,20 @@ public void run() {
 
       if (outputAndStatus.getOutput().isPresent()) {
         ObjectMapper objectMapper = new ObjectMapper();
-        String json = objectMapper.writeValueAsString(outputAndStatus.getOutput().get());
-        setJobOutput(connectionPool, jobId, json);
-        LOGGER.info("Set job output for job " + jobId);
-      } else {
-        LOGGER.info("No output present for job " + jobId);
+        outputAndStatus
+            .getOutput()
```

Comment:
What is `getOutput`? 

If there is an assumption in the code that it is json, then it should be part of the name.

Reply:
per conversation in sync. going to let you handle this as part of your investigation into using json strings in json2pojo.